tests/three_link_it.rs

```diff
@@ -1,40 +1,34 @@
-use crate::event_test_utils::run_simulation_and_compare_events;
-use rust_q_sim::config::{Config, RoutingMode};
+use crate::event_test_utils::{compare_events, run_mpi_simulation_and_convert_events};
 use serial_test::serial;
 
 mod event_test_utils;
 
 #[test]
-#[ignore]
 #[serial]
-fn three_link_network() {
-    let config = Config::builder()
-        .network_file(String::from("./assets/3-links/3-links-network.xml"))
-        .population_file(String::from("./assets/3-links/1-agent.xml"))
-        .output_dir(String::from(
-            "./test_output/controller_it/three_link_network/static",
-        ))
-        .num_parts(1)
-        .start_time(0)
-        .end_time(86400)
-        .build();
-    run_simulation_and_compare_events(config, "tests/resources/three_link")
+fn test_three_link_without_routing() {
+    test_three_link("use-plans", "assets/3-links/1-agent.xml", "static")
 }
 
 #[test]
-#[ignore]
 #[serial]
```

Comment:
Is serial still required? I though you had solved that problem.

Reply:
My fix was for the "old" parallel simulation when we called the controller directly from the test. Now we only call the cargo mpirun command and evaluate when it's finished. 
Locally it works with removing the `serial` but there are 2 things I'm not sure of:
1. is the CI capable of running multiple MPI programs at once since it fails with one program that has 4 processes
2. the output is very messy since tests are run in parallel

```
[2023-02-08 10:40:25.557499 +00:00] INFO [rust_q_sim::mpi::simulation] src/mpi/simulation.rs:61: #0 of Qsim at 7:10
[2023-02-08 10:40:25.558376 +00:00] INFO [rust_q_sim::mpi::simulation] src/mpi/simulation.rs:61: #0 of Qsim at 1:0
[2023-02-08 10:40:25.559002 +00:00] INFO [rust_q_sim::mpi::simulation] src/mpi/simulation.rs:61: #0 of Qsim at 7:20
[2023-02-08 10:40:25.559995 +00:00] INFO [rust_q_sim::mpi::simulation] src/mpi/simulation.rs:61: #0 of Qsim at 1:10
[2023-02-08 10:40:25.560615 +00:00] INFO [rust_q_sim::mpi::simulation] src/mpi/simulation.rs:61: #0 of Qsim at 7:30
[2023-02-08 10:40:25.561593 +00:00] INFO [rust_q_sim::mpi::simulation] src/mpi/simulation.rs:61: #0 of Qsim at 1:20
[2023-02-08 10:40:25.562160 +00:00] INFO [rust_q_sim::mpi::simulation] src/mpi/simulation.rs:61: #0 of Qsim at 7:40
[2023-02-08 10:40:25.563104 +00:00] INFO [rust_q_sim::mpi::simulation] src/mpi/simulation.rs:61: #0 of Qsim at 1:30
```
For 1 I can just try it out. What do you think about 2? I personally prefer a non confusing log thus the `serial` flag.